### PR TITLE
STCOM-865 Apply guidelines for the display of shortcut key combinatio…

### PR DESCRIPTION
**Pull request by CalamityC**
…ns in an app's "Keyboard shortcuts" modal

* present shortcut key combinations in camel case with spaces (guideline 1) in KeyboardShortcutsModal
* add missing default shortcuts
* change description and order of default shortcuts (guideline 2)
* refs ERM-1757, ERM-1758, ERM-1759, ERM-1760, ERM-1761

### CODE REVIEW

File: lib/KeyboardShortcutsModal/KeyboardShortcutsModal.js [hunk 1]
```diff
@@ -27,18 +27,24 @@ function KeyboardShortcutsModal(props) {
   const shortcutsData = [];
```
Comment (aditya-matukumalli) [inline lib/KeyboardShortcutsModal/KeyboardShortcutsModal.js:27]: This piece of code from L27-L49 could probably made a bit more concise.

```suggestion
 const shortcutsData = allCommands.map(({ label, shortcut }) => {
      const shortcutCommand = shortcut.replace(/ /g, '').split('+');
      const commandArray = shortcutCommand.map(cmd => upperFirst(cmd.toLowerCase()));
      let camelCasedShortcut;

      if (platform.includes('Mac')) {
        camelCasedShortcut = commandArray.join(' + ').replace('Mod', 'Cmd').replace('Alt', 'Option');
      } else {
        camelCasedShortcut = commandArray.join(' + ').replace('Mod', 'Ctrl');
      }

      return {
        'action': label,
        'shortcut': camelCasedShortcut,
      };
    });
```

**Review by aditya-matukumalli — CHANGES_REQUESTED** [review]

**Review by aditya-matukumalli — APPROVED** [review]

**Comment by Jagusti** [comment]
@aditya-matukumalli  @JohnC-80 is there something we need to do within ERM team to enable this to progress for merge?

**Comment by aditya-matukumalli** [comment]
> @aditya-matukumalli @JohnC-80 is there something we need to do within ERM team to enable this to progress for merge?

This PR is still awaiting the confirmation of the final list of shortcut keys that need to go into stripes-components. Last I know, Khalilah and Gill are in talks about the final list.

**Comment by aditya-matukumalli** [comment]
Closing this PR. All the changes from this PR has been pulled into the ongoing work in this branch https://github.com/folio-org/stripes-components/tree/export-keyboard-shortcuts-api .

**Pull request closed without merging**